client/src/store/modules/scheduledEvent/state.js

```diff
@@ -2,8 +2,9 @@ export default {
     scheduledEventsFilterView: false,
     scheduledEvents: {},
     eventsPagination: {
-        pageStartEventNum: 0,
-        pageEndEventNum: 0,
-        allEvents: 0
+        currentPage: 1,
+        lastPage: 1,
+        perPage: 1,
+        total: '-'
```

Comment:
If i understood right, total is amount of all events, if it's, change it to `0`, not string